fizzbuuzz.js

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
Weldone but please
- Format your work
- This program doesn't work as it is supposed to, Do you really see `FizzBuzz` if you pass in arrays whose total lengths is a number divisible by both 5 and 3 e.g 15? The problem is with the order of your if statements